src/stats.jl

```diff
@@ -112,7 +112,7 @@ end
 
 function chaindataframe_changerate(name, names_of_params, chains; kwargs...)
     # Compute the change rates.
-    changerates, mvchangerate = chains
+    changerates, mvchangerate = changerate(chains)
```

Comment:
Was this just missing by mistake or was there a reason why `changerate` wasn't there before?

Reply:
No it was a bug but we only noticed it since Julia 1.6 performs some more checks when building `NamedTuple`s. Due to this bug fix I incremented the version number.